packages/plugin-ext/src/main/node/plugin-deployer-impl.ts

```diff
@@ -132,14 +132,15 @@ export class PluginDeployerImpl implements PluginDeployer {
         }
     }
 
-    async deploy(pluginEntry: string, type: PluginType = PluginType.System): Promise<void> {
+    async deploy(pluginEntry: string, type: PluginType = PluginType.System, options?: PluginDeployOptions): Promise<void> {
         const startDeployTime = performance.now();
-        await this.deployMultipleEntries([pluginEntry], type);
+        await this.deployMultipleEntries([pluginEntry], type, options);
         this.logMeasurement('Deploy plugin entry', startDeployTime);
     }
 
-    protected async deployMultipleEntries(pluginEntries: ReadonlyArray<string>, type: PluginType = PluginType.System): Promise<void> {
-        const pluginsToDeploy = await this.resolvePlugins(pluginEntries, type);
+    protected async deployMultipleEntries(pluginEntries: ReadonlyArray<string>, type: PluginType = PluginType.System, options?: PluginDeployOptions): Promise<void> {
+        let pluginsToDeploy = [];
+        pluginsToDeploy = await this.resolvePlugins(pluginEntries, type, options);
```

Comment:
Is this `let` declaration necessary, and if so, why? (Same question re: l. 179 below)